crates/host/src/wasmbus/mod.rs

```diff
@@ -2642,7 +2655,7 @@ impl Host {
         Ok(())
     }
 
-    // TODO(#1548): With actor IDs, new actor references, configuration, etc, we're going to need to do some
+    // TODO(#1548): With component IDs, new actor references, configuration, etc, we're going to need to do some
```

Comment:
Heads up that I don't see this covered in #1548. We should probably create an issue to decide what we're going to do here so we know if it would be a breaking change or not